hydra-core/lib/tasks/hydra.rake

```diff
@@ -3,29 +3,32 @@ require 'active_fedora/rake_support'
 namespace :hydra do
   desc "Start a solr, fedora and rails instance"
   task :server do
-    with_server('development') do
-      # If HOST specified, bind to that IP with -b
-      server_options = " -b #{ENV['HOST']}" if ENV['HOST']
-      IO.popen("rails server#{server_options}") do |io|
-        begin
-          io.each do |line|
-            puts line
+    with_server(ENV['RAILS_ENV'] || 'development') do
+      puts "Fedora: #{ActiveFedora.config.credentials[:url]}"
+      puts "Solr..: #{ActiveFedora.solr_config[:url]}"
+      begin
+        if ENV['SKIP_RAILS']
+          puts "^C to exit"
+          sleep
+        else
+          # If HOST specified, bind to that IP with -b
+          server_options = " -b #{ENV['HOST']}" if ENV['HOST']
+          IO.popen("rails server#{server_options}") do |io|
+            io.each do |line|
+              puts line
+            end
           end
-        rescue Interrupt
-          puts "Stopping server"
         end
+      rescue Interrupt
+        puts "Stopping server"
       end
     end
   end
 
   desc "Start solr and fedora instances for tests"
   task :test_server do
-    with_server('test') do
-      begin
-        sleep
-      rescue Interrupt
-        puts "Stopping server"
-      end
-    end
+    ENV['RAILS_ENV'] ||= 'test'
```

Comment:
should it be `||= 'test'`, or since the task is called `test_server`, should it force the issue with just `= 'test'`. I think i've seen this pattern in other non-hydra places, maybe including rails itself,some tasks force the env.